### DISCUSSION TITLE
MLM2Pro with GS Pro-

### DISCUSSION THREAD

**Discussion by Dragonlord174** [springbok/MLM2PRO-GSPro-Connector#143] [General]
Hello,
I am having issue with short chip shot using MLM2Pro with GS Pro---It might pick up 2 out of 10 short chips-- Trying to understand what is the issue-- It picks up all my shots until I get close to the green. Sometimes the connector picks up the short chip shot but the ball doesn't not move - other times the connector register a failure. Does anyone else have this issue and does anyone know a fix?

Thanks
Dragonlord174

**Comment by wonder99** [comment]
Are you using OCR screenshotting or direct Bluetooth connection?

**Reply by Dragonlord174** [reply]
I use both bluetooth and local internet connection-- both have issue - not using OCR
